src/Gaufrette/Adapter/Phpseclib/tests/bootstrap.php.dist

```diff
@@ -0,0 +1,7 @@
+<?php
+
+$baseDir = dirname(__DIR__);
+
+$loader = require __DIR__.'/../vendor/autoload.php';
+$loader->add('Gaufrette', array($baseDir.'/vendor/knplabs/gaufrette/tests/'));
```

Comment:
use ``autoload-dev`` instead of altering the configuration of the generated loader. this will make this whole file useless